src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -10,7 +10,7 @@
     <Shipping>true</Shipping>
     <PackProject>true</PackProject>
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-    <RuntimeIdentifier>win10-x64</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(DotNetBuildFromSource)' != 'true'">win10-x64</RuntimeIdentifier>
```

Comment:
no clue tbh.

Reply:
The first commit with this file with a RuntimeIdentifiers refers to a commit to ilmerge 
https://github.com/NuGet/NuGet.Client/commit/38949ce93081016167dfd8e37e4e897bedf47d65#diff-3b74a927d8ddbc2c49178f751d0fd058

Note: NuGet.Build.Tasks.Pack.Library.csproj was renamed to NuGet.Build.Tasks.Pack.csproj

Reply:
This RuntimeIdentifier (RID) is here because at ilmerge.exe-time, System.Security.Cryptography.ProtectedData (S.S.C.PD) 4.3.0 only has assets for .NETFramework, not .NET Core. This RID helps ilmerge.exe to find the assembly. Upgrading to S.S.C.PD to 4.4.0+ removes the need of using this RID